packages/sdk/src/api/prover/prover.ts

```diff
@@ -19,8 +19,8 @@ import { v_call } from "./v_call";
 import { v_getProofReceipt } from "./v_getProofReceipt";
 import { foundry } from "viem/chains";
 import { v_versions } from "./v_versions";
-import { checkVersionCompatibility } from "./utils/versions";
-import meta from "../../package.json" with { type: "json" };
+import { checkVersionCompatibility } from "../utils/versions";
+import meta from "../../../package.json" assert { type: "json" };
```

Comment:
<3

Reply:
Actually, `with` is the new syntax and I should not have to change it. Let me revert this change.